cluster/gce/util.sh

```diff
@@ -1046,8 +1056,9 @@ function prepare-push() {
 
     # Ugly hack: Since it is not possible to delete instance-template that is currently
     # being used, create a temp one, then delete the old one and recreate it once again.
-    create-node-instance-template "tmp"
+    create-node-instance-template "${NODE_INSTANCE_PREFIX}-template-tmp"
```

Comment:
I'd create another local variable (`template_name_tmp`) for this as well, and use it below, so things are all in one place.